README.md

```diff
@@ -40,24 +40,34 @@ The following SQL can be used to create the `imrt_ingest` and `imrt_search` user
 CREATE USER imrt_ingest WITH PASSWORD '[choose a password]';
 CREATE USER imrt_search WITH PASSWORD '[choose a password]';
 
-REVOKE CONNECT ON DATABASE imrt FROM PUBLIC;
-
+GRANT CONNECT ON DATABASE imrt TO imrt_admin;
```

Comment:
I'm probably behind on the changes you made last week, but isn't imrt_admin the super user when you create the database instance in AWS? And the user that is running this script? If so, then do we need to grant them stuff in here?

Reply:
Strictly speaking, the script doesn't necessarily have to be run as `imrt_admin`; it could be run as _any_ super user (i.e. a user with the privileges necessary to create database objects).  I added the `GRANT CONNECT` in case someone creates the database(s) as another user (even though the instructions are explicit about using `imrt_admin`).

Having said all that, I think since the `REVOKE CONNECT... FROM PUBLIC` line has been removed, the explicit `GRANT` to `imrt_admin` can probably be removed.